docs/mkdocs/docs/integration/build2/manifest

```diff
@@ -0,0 +1,14 @@
+name: example
+version: 0.1.0-a.0.z
+language: c++
+summary: example C++ executable
+license: other: proprietary ; Not free/open source.
+description-file: README.md
+url: https://example.org/example
+email: your@emailprovider.com
+#build-error-email: your@emailprovider.com
+depends: * build2 >= 0.16.0
+depends: * bpkg >= 0.16.0
+#depends: libhello ^1.0.0
```

Comment:
This line can be removed, right?

Reply:
Yes its a comment generated by `bdep new` as an example to how to add package dependencies. Following the instructions using `bdep new`, that comment will appear in the file, but it can be removed.

Reply:
Ok, then leave it as is.